internal/provider/resource_saml_idp.go

```diff
@@ -229,6 +230,7 @@ func resourceWizSAMLIdPRead(ctx context.Context, d *schema.ResourceData, m inter
 	        allowManualRoleOverride
 	        certificate
 	        domains
+			issuerURL
```

Comment:
Indentation on the graphql query is a tab for code indentatoin, followed by spaces for query indentation.  To keep things consistent, this formatting should be maintained.